### PR TITLE
Fix part of #632 : Replace current recyclerview implementation with BindableAdapter usage in Help Activity

### PR DESCRIPTION
**Pull request by SayantanBanerjee16**
## Explanation

Fixes part of #632 
Bindable Adapter has been introduced to Help Activtity.
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/help/HelpListViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  val helpItemItemList: List<HelpItemViewModel> by lazy {
+    getRecyclerViewItemList()
+  }
+
+  private fun getRecyclerViewItemList(): ArrayList<HelpItemViewModel> {
+    for (item in HelpItems.values()) {
+      if (item == HelpItems.FAQ) {
+        val category1 = activity.getString(R.string.frequently_asked_questions_FAQ)
+        val helpViewModel = HelpItemViewModel(activity,category1)
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/help/HelpListViewModel.kt]: Nit spacing issue:
`val helpViewModel = HelpItemViewModel(activity, category1)`

**Review by rt4914 — APPROVED** [review]
Great. LGTM, just one nit change suggested.

File: app/src/main/java/org/oppia/app/help/HelpListViewModel.kt (hunk 1 above)
Comment (SayantanBanerjee16) [reply app/src/main/java/org/oppia/app/help/HelpListViewModel.kt]: Done.

**Comment by SayantanBanerjee16** [comment]
@rt4914 Done with the changes. Please merge. Thanks :)

**Pull request merged**